### DISCUSSION TITLE
冷门FTP的自定义链接错误

### DISCUSSION THREAD

**Discussion by qvsijia** [Qexo/Qexo#186] [General]
保存路径：/htdocs/gallery/{time}.{extName}
自定义链接：http://qvsijia.unaux.com/

上传成功，没有问题

但是Qexo显示的链接是：http://qvsijia.unaux.com/htdocs/gallery/{time}.{extName}
实际上真正的链接是：http://qvsijia.unaux.com/gallery/{time}.{extName}

**Comment by am-abudu** [comment]
建议修改ftp配置文件 因为从逻辑上讲qexo的逻辑没有问题

**Comment by am-abudu** [comment]
在下一个版本将要求填写完整path以规避该问题

**Comment by am-abudu** [comment]
3746e96f6ead28c6600ff89b07d5af9ecafe0e85
